### ISSUE TITLE
`CHE_DEVFILE_REGISTRY_URL` is not properly defined on the OpenShift template level

### ISSUE THREAD

**Issue by ibuziuk** [eclipse-che/che#19084]
### Describe the bug
`CHE_DEVFILE_REGISTRY_URL` is only defined on the helm chart, but not on OpenShift template

This results in failure on entrypoint level - https://github.com/eclipse/che-devfile-registry/blob/ed4990f6c9293fc79d22a47a29eda510e9130fbb/build/dockerfiles/entrypoint.sh#L142-L152 

for plugins that refer to ``{{ DEVFILE_REGISTRY_URL }}`` - https://github.com/eclipse/che-devfile-registry/blob/dcfdacf3954e30a25a8af3c7bd497b193d24f14a/devfiles/quarkus/devfile.yaml#L60

```
if grep -q '{{ DEVFILE_REGISTRY_URL }}' "${devfiles[@]}"; then
```

It looks like `CHE_DEVFILE_REGISTRY_URL` is not properly defined for OpenShift case

### Che version
7.26.0 

### Installation method
- OpenShift template

### Environment
OSD 4

### Eclipse Che Logs
Production deployment of the registry failed with:

````
Checking devfile /var/www/html/devfiles/angular/devfile.yaml
Checking devfile /var/www/html/devfiles/apache-camel-k/devfile.yaml
Checking devfile /var/www/html/devfiles/apache-camel-springboot/devfile.yaml
Checking devfile /var/www/html/devfiles/bash/devfile.yaml
Checking devfile /var/www/html/devfiles/che4z/devfile.yaml
Checking devfile /var/www/html/devfiles/cpp/devfile.yaml
Checking devfile /var/www/html/devfiles/dotnet/devfile.yaml
Checking devfile /var/www/html/devfiles/dotnet-asp.net/devfile.yaml
Checking devfile /var/www/html/devfiles/go/devfile.yaml
Checking devfile /var/www/html/devfiles/java-gradle/devfile.yaml
Checking devfile /var/www/html/devfiles/java-maven/devfile.yaml
Checking devfile /var/www/html/devfiles/java-mongo/devfile.yaml
Checking devfile /var/www/html/devfiles/java-mysql/devfile.yaml
Checking devfile /var/www/html/devfiles/java-web-spring/devfile.yaml
Checking devfile /var/www/html/devfiles/java-web-vertx/devfile.yaml
Checking devfile /var/www/html/devfiles/nodejs/devfile.yaml
Checking devfile /var/www/html/devfiles/nodejs-mongo/devfile.yaml
Checking devfile /var/www/html/devfiles/nodejs-react/devfile.yaml
Checking devfile /var/www/html/devfiles/nodejs-yarn/devfile.yaml
Checking devfile /var/www/html/devfiles/php-laravel/devfile.yaml
Checking devfile /var/www/html/devfiles/php-mysql/devfile.yaml
Checking devfile /var/www/html/devfiles/php-symfony/devfile.yaml
Checking devfile /var/www/html/devfiles/php-web-simple/devfile.yaml
Checking devfile /var/www/html/devfiles/python/devfile.yaml
Checking devfile /var/www/html/devfiles/python-django/devfile.yaml
Checking devfile /var/www/html/devfiles/quarkus/devfile.yaml
Checking devfile /var/www/html/devfiles/quarkus-command-mode/devfile.yaml
Checking devfile /var/www/html/devfiles/rust/devfile.yaml
Checking devfile /var/www/html/devfiles/scala-sbt/devfile.yaml
WARNING: environment variable 'CHE_DEVFILE_REGISTRY_URL' not configured for an offline build of this registry. This may cause issues with importing projects in a workspace.
sed: -e expression #1, char 48: unterminated `s' command
```

### Additional context
In order to workaround the issue the following PR was provided - https://github.com/eclipse/che-devfile-registry/pull/337

**Comment by ericwill** [comment]
cc @tolusha

**Comment by tolusha** [comment]
It seems so. 
I don't see `CHE_DEVFILE_REGISTRY_URL` defined in OpenShift templates

**Comment by vitaliy-guliy** [comment]
I checked today how it works with operator and didn't get any problems.

From the other side, if the variable is not set, everything should be working just because the URL for the replacement is formed from `SERVICE_HOST` and `SERVICE_PORT`.

https://github.com/eclipse-che/che-devfile-registry/blob/main/build/dockerfiles/entrypoint.sh#L151

Could anyone check this issue again to be sure, that it's not relevant anymore?
(I believe, that something has been changed, that affected on this case)
